.github/workflows/run-multi-lang-tests.yml

```diff
@@ -0,0 +1,194 @@
+        description: 'Authentication password'
+      timeout-minutes:
+        required: false
+        type: number
+        default: 30
+        description: 'Job timeout in minutes'
+      artifact-is-tarball:
+        required: false
+        type: boolean
+        default: false
+        description: 'Whether the artifact is a tarball (tar.gz) that needs to be extracted'
+
+jobs:
+  run-tests:
+    name: Run Multi-language SDK Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: ${{ inputs.timeout-minutes }}
+    steps:
+      - name: Checkout greptimedb-tests repository
+        uses: actions/checkout@v4
+        with:
+          repository: GreptimeTeam/greptimedb-tests
+          persist-credentials: false
+
+      - name: Download pre-built greptime binary
+        uses: actions/download-artifact@v4
+        with:
+          name: ${{ inputs.artifact-name }}
+          path: artifact
+
+      - name: Setup greptime binary
+        run: |
+          mkdir -p bin
+          if [ "${{ inputs.artifact-is-tarball }}" = "true" ]; then
+            # Extract tarball and find greptime binary
+            tar -xzf artifact/*.tar.gz -C artifact
+            find artifact -name "greptime" -type f -exec cp {} bin/greptime \;
+          else
+            # Direct binary format
+            if [ -f artifact/greptime ]; then
+              cp artifact/greptime bin/greptime
+            else
+              cp artifact/* bin/greptime
+            fi
+          fi
+          chmod +x ./bin/greptime
+          ls -lh ./bin/greptime
+          ./bin/greptime --version
+
+      - name: Setup Java 17
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+          cache: 'maven'
+
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.8'
+
+      - name: Setup Go 1.24
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.24'
+          cache: true
+          cache-dependency-path: go-tests/go.sum
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+
+      - name: Install Python dependencies
+        run: |
+          pip install mysql-connector-python psycopg2-binary
+          python3 -c "import mysql.connector; print(f'mysql-connector-python {mysql.connector.__version__}')"
+          python3 -c "import psycopg2; print(f'psycopg2 {psycopg2.__version__}')"
+
+      - name: Install Go dependencies
+        working-directory: go-tests
+        run: |
+          go mod download
+          go mod verify
+          go version
+
+      - name: Kill existing GreptimeDB processes
+        run: |
+          pkill -f greptime || true
+          sleep 2
+
+      - name: Start GreptimeDB standalone
+        run: |
+          ./bin/greptime standalone start \
+            --http-addr 0.0.0.0:${{ inputs.http-port }} \
+            --rpc-addr 0.0.0.0:4001 \
```

Comment:
Way not use `0.0.0.0:${{ inputs.rpc-port }}`?

Reply:
Good catch!